pkg/model/dev.go

```diff
@@ -230,8 +230,8 @@ type Volume struct {
 
 // Sync represents a sync info in the development container
 type Sync struct {
-	LocalPath      string
-	RemotePath     string
+	LocalPath      string       `json:"-"`
```

Comment:
I don't think we use json for this struct at all. Should we change it to yaml or add it to the tags?

Reply:
agreed, changed to cd97c3a9